test/xrt/01_air_to_npu/run_peano.lit

```diff
@@ -0,0 +1,7 @@
+// (c) Copyright 2023 Advanced Micro Devices, Inc.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
mlir-air is MIT license. I see the same bug in the cut-and-paste source.

Reply:
Thanks for catching that! This is really helpful.

I fixed all lit test files touched in this PR, but will fix other source files in a separate PR.